src/extension.ts

```diff
@@ -28,6 +29,7 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
     context.subscriptions.push(ext.outputChannel);
 
     registerAppServiceExtensionVariables(ext);
+    registerAzureUtilsExtensionVariables(ext);
```

Comment:
Does the order of these three calls matter?

Reply:
In this order, `registerAzureUtilsExtensionVariables` does nothing so we could remove it or swap the order. To be verbose I'll swap the order.